Frontend/CO.CDP.OrganisationApp/Helpers/ApiHelper.cs

```diff
@@ -0,0 +1,30 @@
+using CO.CDP.Organisation.WebApiClient;
+
+namespace CO.CDP.OrganisationApp.Helpers;
+
+public static class ApiHelper
+{
+    public static async Task<T> CallApiAsync<T>(Func<Task<T>> apiCall, string errorMessage)
```

Comment:
This is only handling errors from Organisation API only, are why are we rethrowing them again.

Reply:
After our discussion, I have removed it now.